doc/source/appendixa.rst

```diff
@@ -103,7 +103,7 @@ Considering that the entire atlas occupies close to 70 MB of disk space,
 many applications could be satisfied by a copy of the solar metallicity
 spectra only (Table 2 of the README file).
 See
-`Kurucz 1993 atlas README file <http://ssb.stsci.edu/cdbs/grid/k93models/AA_README>`_
+`Kurucz 1993 atlas README file <ftp://ftp.stsci.edu/cdbs/grid/k93models/AA_README>`_
```

Comment:
Please revert all `ssb.stsci.edu -> ftp`. Some Macs don't support FTP anymore. The HTTP is a workaround.

Reply:
@pllim Will that http link exist after Monday?

Reply:
The link text [here](https://pysynphot.readthedocs.io/en/latest/how_works.html#the-pysynphot-database) to `ssb.stsci.edu/cdbs` does not return a valid site.  Did you have an alternative?

Reply:
Just `ssb.stsci.edu/cdbs` is not supposed to work. You need the URL to the exact file, like http://ssb.stsci.edu/cdbs/grid/k93models/AA_README .